### PR TITLE
MDSS-4021 - Support for POSTGRES on Instance creation

### PR DESCRIPTION
**Pull request by avenkatram85**
Support for POSTGRES on Instance creation

### CODE REVIEW

File: client/mds/controller/service.go [hunk 1]
```diff
@@ -151,6 +151,11 @@ func (s *Service) GetServiceInstanceTypes(serviceTypeQuery *MdsInstanceTypesQuer
 	reqUrl := fmt.Sprintf("%s/%s/%s", s.Endpoint, Services, InstanceTypes)
 	var response model.MdsInstanceTypeList
 
+	serviceTypeQuery.ServiceType = serviceTypeQuery.ServiceType
```
Comment (sharmata-vmw) [inline client/mds/controller/service.go:154]: is it a mistake?

Comment (avenkatram85) [reply client/mds/controller/service.go:154]: Its the previous commit

File: mds/data_source_instance_types.go [hunk 2]
```diff
@@ -125,25 +127,59 @@ func (d *instanceTypesDataSource) Schema(_ context.Context, _ datasource.SchemaR
...
+
+		parsedNodes, err := strconv.ParseInt(instanceType.Metadata.Nodes, 10, 64)
+		if err != nil {
+			resp.Diagnostics.AddError(
+				"Failed to parse Nodes value",
+				err.Error(),
+			)
+			return
+		}
+		nodes = parsedNodes
```
Comment (sharmata-vmw) [inline mds/data_source_instance_types.go:171]: we can eliminate these middleman variable assigment, use original declared above

File: client/mds/controller/service.go (hunk 1 above)
Comment (sharmata-vmw) [reply client/mds/controller/service.go:154]: okay

**Review by sharmata-vmw — APPROVED** [review]

File: mds/data_source_instance_types.go (hunk 2 above)
Comment (avenkatram85) [reply mds/data_source_instance_types.go]: Addressed

**Pull request merged**